added new requirements (#3)
mate-academy/jv-pecs: README.md

1. There is also MachineService interface created. You need to parameterize it as well and replace `Object` in method signature with the right option (use PECS):

    - the method `getAll(Class type)` produces the list of machines based on the input param.

>     - the method `fill(List<Object> machines, Object value)` fill the machines list with passed value.

Author: це назва методу. Я б його так і залишив. Метод є аналогом https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#fill(java.util.List,%20T)

>     - the method `fill(List<Object> machines, Object value)` fill the machines list with passed value.

Reviewer: сори, я не там дописала "s"

>     - the method `fill(List<Object> machines, Object value)` fill the machines list with passed value.

Reviewer: ```suggestion
    - the method `fill(List<Object> machines, Object value)` fills the machines list with passed value.
```

    - the method `startWorking()` should be able to accept a list containing any Machine.

1. Create a class implementing MachineService and realize these methods:

> - `getAll(Class type)`: based on the input class type, choose the right MachineProducer implementation from those you created earlier, and return the result of its `get()` method.

Reviewer: ```suggestion
- `getAll(Class type)`: based on the input class type, choose the right MachineProducer implementation from those you created earlier, and call its `get()` method.
```

For example: `if (type == Bulldozer.class)` - we should call the `get()` method from right implementation of MachineProducer (the one that will return `List<Bulldozer>`) and return these machines.

> 

Reviewer: ```suggestion

- `fill()` - add the passed value (which can be of any Machine  subtype) to the passed list

```
